tests/perf-regression-user-profiles.yaml

```diff
@@ -41,11 +41,11 @@ backends: !mux
         gce_instance_type_loader: 'n1-standard-2'
         gce_root_disk_type_loader: 'pd-standard'
         gce_n_local_ssd_disk_loader: 0
-        gce_instance_type_monitor: 'n1-standard-1'
+        gce_instance_type_monitor: 'n1-standard-2'
         gce_root_disk_type_monitor: 'pd-standard'
         gce_root_disk_size_monitor: 50
         gce_n_local_ssd_disk_monitor: 0
-        scylla_repo: 'https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/branch-1.7/17/scylla.repo'
+        scylla_repo: 'REPO_FILE_PATH'
```

Comment:
Good idea to replace the fixed version (or other parameters), it's easy to out of data, sometimes we forget to update the version (or other info).

Can we use same prefix for all parameters that need to be modified before execution? such as CONF_REPO_FILE_PATH or PARAM_ REPO_FILE_PATH, or CUSTOM_REPO_FILE_PATH, then we can check the yaml file in Jenkins Shell or inside SCT (not good).

Reply:
Yes, we can.
I think PARAM is a good option, I'll add it as a prefix.